src/index.html

```diff
@@ -12,6 +12,40 @@
 </head>
 
 <body>
+  <nav class="navbar-institutional navbar navbar-expand-lg navbar-dark">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsTop" aria-controls="navbarsTop"
+        aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div>
+        <a class="navbar-brand" href="http://www.agid.gov.it/">AgID</a>
+        <span class="navbar-brand text-white"> + </span>
+        <a class="navbar-brand" href="https://teamdigitale.governo.it/">Team
+            <span class="d-none d-lg-inline">per la Trasformazione </span>Digitale</a>
+    </div>
+    <div class="collapse navbar-collapse" id="navbarsTop">
+        <ul class="navbar-nav ml-auto">
+            <li class="nav-item nav-item-w-divider">
+                <a class="nav-link" href="https://pianotriennale-ict.italia.it/">Piano Triennale</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="https://developers.italia.it/">Developers</a>
+            </li>
+            <li class="nav-item nav-item-w-divider">
+                <a class="nav-link" href="http://designers.italia.it/">Designers</a>
```

Comment:
Molto probabilmente è sbagliato già nel progetto da cui l'hai preso, ma potresti usare https anche per questo link (`https://designers.italia.it/`)? Grazie